rb_ws/src/buggy/scripts/2d_sim/controller_2d.py

```diff
@@ -1,4 +1,5 @@
 #! /usr/bin/env python3
+import sys
```

Comment:
Why do we need this?

Reply:
For taking command line arguments, since the buggy name is now passed via command line to controller_2d